core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -389,10 +391,61 @@ private static MediaType addKnownType(MediaType mediaType) {
     public static final MediaType FORM_DATA =
             createConstant(APPLICATION_TYPE, "x-www-form-urlencoded");
 
+    /**
+     * A {@link MediaType} constant representing {@code multipart/alternative} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc1521.txt">RFC 1521:
+     * MIME Part One: Mechanisms for Specifying and Describing the Format of Internet Message Bodies</a>
+     */
+    public static final MediaType MULTIPART_ALTERNATIVE = createConstant(MULTIPART_TYPE, "alternative");
+
+    /**
+     * A {@link MediaType} constant representing {@code multipart/digest} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc1521.txt">RFC 1521:
+     * MIME Part One: Mechanisms for Specifying and Describing the Format of Internet Message Bodies</a>
+     */
+    public static final MediaType MULTIPART_DIGEST = createConstant(MULTIPART_TYPE, "digest");
+
+    /**
+     * A {@link MediaType} constant representing {@code multipart/encrypted} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc1847.txt">RFC 1847:
+     * Security Multiparts for MIME: Multipart/Signed and Multipart/Encrypted</a>
+     */
+    public static final MediaType MULTIPART_ENCRYPTED = createConstant(MULTIPART_TYPE, "encrypted");
+
     /**
      * A {@link MediaType} constant representing {@code multipart/form-data} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc1867.txt">RFC 1867:
+     * Form-based File Upload in HTML</a>
+     */
+    public static final MediaType MULTIPART_FORM_DATA = createConstant(MULTIPART_TYPE, "form-data");
+
+    /**
+     * A {@link MediaType} constant representing {@code multipart/mixed} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc1521.txt">RFC 1521:
+     * MIME Part One: Mechanisms for Specifying and Describing the Format of Internet Message Bodies</a>
      */
-    public static final MediaType MULTIPART_FORM_DATA = createConstant("multipart", "form-data");
+    public static final MediaType MULTIPART_MIXED = createConstant(MULTIPART_TYPE, "mixed");
+
+    /**
+     * A {@link MediaType} constant representing {@code multipart/parallel} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc1521.txt">RFC 1521:
+     * MIME Part One: Mechanisms for Specifying and Describing the Format of Internet Message Bodies</a>
+     */
+    public static final MediaType MULTIPART_PARALLEL = createConstant(MULTIPART_TYPE, "parallel");
+
+    /**
+     * A {@link MediaType} constant representing {@code multipart/related} media type.
+     * As described in <a href="https://datatracker.ietf.org/doc/html/rfc2112.txt">RFC 2112:
```

Comment:
Could you remove `.txt` from all HTML RFC links?

Reply:
removed